### DISCUSSION TITLE
Artillery Cloud is GA

### DISCUSSION THREAD

**Discussion by hassy** [artilleryio/artillery#2780] [Announcements]
Artillery Cloud is now out of beta and is generally available.

→ Sign up with your GitHub account on https://www.artillery.io

https://github.com/artilleryio/artillery/assets/1490/198285c3-57a2-4065-bfa1-f353f45f58b3

## Features

With Artillery Cloud you can:

* See summary reports of key performance metrics in your load tests:
  * response times
  * request rates
  * errors
  * custom success conditions
  * custom metrics
* See a green/red overview of all load testing activity on your team
* Create custom charts and dashboards to help make sense of application-specific metrics
* Annotate and comment on individual test reports to include debugging information or results of performance investigations
* Share tests with external stakeholders and other teams

### First-class support for Playwright

Artillery Cloud has first-class support for load testing with Playwright, including:

* Tailor-made visualizations to show browser performance metrics and their effect on user experience
* Built-in recording & playback of Playwright traces to make it easier to debug test failures

**Comment by Rio517** [comment]
This looks awesome!

![Billing · Settings · Artillery](https://github.com/artilleryio/artillery/assets/14241/0bc42964-917f-42e7-bd0e-fea9b342bcfa)
It would be good if there were a marketing page indicating what the billing plans were!

Thanks!

**Reply by hassy** [reply]
It's coming at some point soon :)
